public/data/Sangjin.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Sangjin",
+  "type": "personal",
+  "bio": "Open Source Enthusiast! & Backend developer",
```

I think we should remove the `!` here 😅 What do you think?
```suggestion
  "bio": "Open Source Enthusiast & Backend developer",
```